app/src/main/java/com/mxt/anitrend/view/activity/detail/ProfileActivity.java

```diff
@@ -60,6 +60,12 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             id = getIntent().getLongExtra(KeyUtil.arg_id, -1);
         if(getIntent().hasExtra(KeyUtil.arg_userName))
             userName = getIntent().getStringExtra(KeyUtil.arg_userName);
+        if(getIntent().hasExtra(KeyUtil.arg_mediaType)) {
```

Comment:
sidenote: It'd be possible to display the list directly but I think it's pretty neat to be able to go back to user profile.

Reply:
Sensible suggestion